errors/doc.go

```diff
@@ -37,6 +37,7 @@ The index of codes are listed below:
 	    5100: NoKeyUsages
 	    5200: InvalidPolicy
 	    5300: InvalidRequest
+	    5400: UnknownProfileLabel
```

Comment:
Can you add this error to [`/doc/errorcode.txt`](https://github.com/cloudflare/cfssl/blob/master/doc/errorcode.txt) too?